tools/plugin-release

```diff
@@ -409,11 +420,19 @@ def prepare(rel_name, archive):
 
     print('Adding vendor libraries')
     if os.path.exists(os.path.join(build_dir, 'composer.json')):
-        composer = ['composer', 'install', '-o', '--no-dev']
+
+        composer_cmd = 'composer'
+        if os.name=='nt':
+            if not cmd_exists([composer_cmd, '--version']):
+                composer_cmd = 'composer.bat'
+        if not cmd_exists([composer_cmd, '--version']):
+            print_err('composer not found!')
+            sys.exit(1)
```

Comment:
I do not really know how `os.name` is computed, but I do not like relying on such stuffs (maybe due to unstabillity of such information on Javascript env).

```python
composer_cmd = 'composer'
if not cmd_exists([composer_cmd, '--version']):
    composer_cmd = 'composer.bat'
    if not cmd_exists([composer_cmd, '--version']):
        print_err('composer not found!')
        sys.exit(1)
```

Reply:
Relying on OS name is indeed not required